.github/workflows/build-steps.yml

```diff
@@ -95,6 +95,10 @@ jobs:
     container:
       image: ${{ inputs.container }}
       volumes: ${{ fromJson( inputs.container_volumes ) }}
+      options: -v /:/host/root ${{ (contains(inputs.runner, 'gpu') && '-e NVIDIA_DRIVER_CAPABILITIES=compute,graphics,utility --gpus all') || '-e A=x' }}
+      # Extra options:
```

Comment:
Not used this conditional syntax before, I'm guessing that the `|| '-e A=x'` part just sets an innocuous env var `A=x` since you can't have an empty "else" part of the conditional?